docs/nx-cloud/private/ami-setup.md

```diff
@@ -0,0 +1,143 @@
+
+secret:
+  # set your initial admin password for logging into the app
+  adminPassword: 'correcthorsebatterystaple'
+```
+
+2. Apply the configuration:
+
+```bash
+scp -r ./myconfiguration.yaml nx-cloud@<your-instance-ip>:~/config/user/update.yaml
+```
+
+That's it! After a few minutes, you should be able to log-in using your password!
+
+### Connecting to your NxCloud instance
+
+In your Nx workspace, you can enable NxCloud by running:
+
+```bash
+NX_CLOUD_API="https://nx-cloud.on.my-domain.ca" npx nx connect
+```
+
+### Advanced configuration and auth
+
+You can optionally enable authentication using your preferred SSO provider:
+
+- GitHub
+- Bitbucket
+- GitLab
+- SAML (Okta, Azure AD etc.)
+
+```yaml
+# This is all you need to get the baseline of your nx-cloud instance configured!
+
+# only use this if you'd like to use any of the newer NxCloud version from here: https://nx.dev/nx-cloud/reference/release-notes#docker-containers
+# global.imageTag: ''
+
+# Set the external URL your instance is running on
+nxCloudAppURL: 'https://nx-cloud.on.my-domain.ca'
+
+# Uncomment (along with github secrets below) to enable working with GitHub pull requests or github auth
+#github:
+#  auth:
+#    enabled: false
+
+#gitlab:
+#  apiUrl: '' # this is only needed if you have a self-hosted instance
+#  auth:
+#    enabled: false
+
+#bitbucket:
+#  auth:
+#    enabled: false
+
+#saml:
+#  auth:
+#    enabled: false
+
+# Provide plaintext values for your application to use. We will extract them,
+# store them within the application runtime, and scrub the plaintext ones from
+# the filesystem
+secret:
+  # set your initial admin password for logging into the app
+  # see here: https://nx.dev/nx-cloud/private-cloud/auth-single-admin
+  adminPassword: 'correcthorsebatterystaple'
+
+  # If you want to enable GitHub Login, just provide your client id & secret, we handle the rest
+  # see here: https://nx.dev/nx-cloud/private-cloud/auth-github
+  githubAuthClientId: 'my_client_id'
+  githubAuthClientSecret: 'my_client_secret'
+
+  # The same goes for GitLab authentication
+  # see here: https://nx.dev/nx-cloud/private-cloud/auth-gitlab
+  # gitlabAppId: 'my_gitlab_app_id'
+  # gitlabAppSecret: 'my_gitlab_app_secret'
+
+  # Bitbucket too! If these are uncommented, BB auth is automatically enabled
+  # see here: https://nx.dev/nx-cloud/private-cloud/auth-bitbucket
+  # bitbucketAppId: 'bitbucket_app_id'
+  # bitbucketAppSecret: 'bitbucket_app_secret'
+
+  # SAML auth
+  # see here: https://nx.dev/nx-cloud/private-cloud/auth-saml
+  # samlEntryPoint: 'your_saml_entry_point'
+  # samlCert: 'saml_cert'
+```
+
+### Upgrades
+
+We send out emails with every new NxCloud release to all our Enterprise customers:
+
+1. You can view your current version at the `/version` route: https://your-nx-cloud-url.com/version
+2. [And these are the latest NxCloud releases](https://nx.dev/nx-cloud/reference/release-notes#docker-containers)
```

Comment:
recomment linking to the top of the page and not part of the way down. 
```suggestion
2. [And these are the latest NxCloud releases](https://nx.dev/nx-cloud/reference/release-notes)
```

Reply:
the top of the page shows releases for the NXCloud runner. In this case, I want to direct to the Docker image versions.

Reply:
Ah okay 👍